app/src/main/res/menu/bibleview_selection.xml

```diff
@@ -53,6 +53,12 @@
 			android:visible="false"
 			app:showAsAction="always"
 			/>
+		<item android:id="@+id/web_search"
+			android:title="@string/web_search"
+			android:orderInCategory="2"
+			app:showAsAction="always"
+			android:visible="false"
```

Comment:
Hmm, looks like this is always hidden. did you try this at all yourself?

Reply:
sorry - I didn't get a chance to build/test

I thought this meant it was only visible in the overflow section of the selection menu - does it not dispkay the web search at all??

Thanks for merging these!